asciidoctorj-cli/src/test/java/org/asciidoctor/cli/WhenAsciidoctorIsCalledUsingCli.java

```diff
@@ -274,6 +275,23 @@ void should_convert_to_subdirectories(@ClasspathResource("relative/sub/test.adoc
         expectedFile.delete();
     }
 
+    @Test
+    void should_print_version() throws IOException {
+        PrintStream previousSystemOut = System.out;
+        try (ByteArrayOutputStream out = new ByteArrayOutputStream();
+            PrintStream sysout = new PrintStream(out)){
+            System.setOut(sysout);
+            new AsciidoctorInvoker().invoke("--version");
+            String result = out.toString(StandardCharsets.UTF_8);
+            Assertions.assertThat(result)
+                    // Needs to be updated when version changes
```

Comment:
Upstream tests fail on line 288, probabley related to the hardcoded assert. Maybe validate simply with a regex or read `asciidoctorj-version.properties` from classpath to assert it's the same?

PS1: Btw, since `createVersionFile` is run during `jar` I don't understand what file is being read during tests but tests work 🤔